va/va_enc_vp9.h

```diff
@@ -151,11 +151,14 @@ typedef struct  _VAEncSequenceParameterBufferVP9
  */
 typedef struct  _VAEncPictureParameterBufferVP9
 {
-    /** VP9 encoder may support dynamic scaling function.
+    /** VP9 encoder supports dynamic scaling function.
      *  If enabled (enable_dynamic_scaling is set), application may request
```

Comment:
I'm confused.
Are we supposed to set separate frame_width_src/dst & frame_height_src/dst only when enabling Bitrate control?
I don't think that assumption is true.
For eg: Spatial encode with CQP mode can just feed width_src/dst & height_src/dst where there is no VAEncMiscParameterRateControl. It used to work(partially) with i965 driver.

Reply:
No, it's not depend on rate control mode. This only allows per frame dynamic resolution change and driver will scale reference frames on the fly if dynamic scaling is enabled. Otherwise resolution can be changed with reset and adding key frame.